rocAL/rocAL_pybind/amd/rocal/plugin/generic.py

```diff
@@ -22,66 +22,78 @@
 import numpy as np
 import rocal_pybind as b
 import amd.rocal.types as types
-import ctypes
+
 
 class ROCALGenericIterator(object):
-    def __init__(self, pipeline, tensor_layout = types.NCHW, reverse_channels = False, multiplier = [1.0,1.0,1.0], offset = [0.0, 0.0, 0.0], tensor_dtype=types.FLOAT, display=False, device="cpu", device_id =0):
+    def __init__(self, pipeline, tensor_layout=types.NCHW, reverse_channels=False, multiplier=[1.0, 1.0, 1.0], offset=[0.0, 0.0, 0.0], tensor_dtype=types.FLOAT, display=False, device="cpu", device_id=0):
```

Comment:
This line change just removed all the spacing for default arguments. I think having spacing like before is better and keep the same coding style everywhere

Reply:
We checked with google coding style guidelines for python and changed the spacing accordingly.
https://google.github.io/styleguide/pyguide.html#3194-default-values